serde/msgpack.py

```diff
@@ -26,9 +32,17 @@ def deserialize(cls, s, raw=False, use_list=False, **opts):
 
 
 def to_msgpack(obj: Any, se=MsgPackSerializer, **opts) -> bytes:
+    "If `ext_dict` option is specified, `obj` is encoded as a `msgpack.ExtType`"
     return se.serialize(obj, **opts)
 
 
 def from_msgpack(c: Type[T], s: str, de=MsgPackDeserializer, named=True, **opts) -> Type[T]:
-    fromf = from_dict if named else from_tuple
-    return fromf(c, de.deserialize(s, **opts))
+    "If `ext_dict` option is specified, `c` is ignored and type is inferred from `msgpack.ExtType`"
```

Comment:
Let's make it a docstring!
```suggestion
    """
    If `ext_dict` option is specified, `c` is ignored and type is inferred from `msgpack.ExtType`
    """"
```

Reply:
Ah I thought it was ok to use single quotes for one line docstrings. But you're right pep-257 recommends triple quotes.